### PR TITLE
Madison/fix visible gpus

### PR DESCRIPTION
**Pull request by madisonmay**

### CODE REVIEW

File: finetune/base.py [hunk 1]
```diff
@@ -204,17 +204,29 @@ def finetune(self, Xs, Y=None, batch_size=None):
...
 
+    def _distribute_strategy(self, visible_gpus):
+        if isinstance(visible_gpus, (list, tuple)):
+            visible_gpus = all_gpus(visible_gpus=tuple(visible_gpus))
+        else:
+            visible_gpus = all_gpus()
+
+        num_gpus = len(visible_gpus)
+        if num_gpus > 1:
+            distribute_strategy = PatchedParameterServerStrategy(num_gpus_per_worker=num_gpus)
```
Comment (madisonmay) [inline finetune/base.py]: Patch attribute inside of PatchedParameterServerStrategy

File: finetune/base.py [hunk 2]
```diff
@@ -57,8 +58,8 @@ def cleanup():
 
         atexit.register(cleanup)
 
-        self.config = config or get_default_config()
-        self.config.update(kwargs)
+        self.config = get_config(**kwargs)
```
Comment (benleetownsend) [inline finetune/base.py]: I don't understand why this changed?

Comment (madisonmay) [reply finetune/base.py]: It doesn't need to be, I should revert this.  At the very least this change should be in a separate PR.

**Comment by madisonmay** [comment]
Bonus: got rid of a keyword argument we no longer use (verbose)

**Pull request merged**